example/index.ts

```diff
@@ -0,0 +1,75 @@
+import { ArangoDBAdapter, Project } from "cruddl"
+import { ApolloServer } from "apollo-server"
+
+// arangodb --starter.local --starter.mode=single  --starter.port=8529
+const db = new ArangoDBAdapter({
+  databaseName: "test",
+  url: "http+tcp://root:@localhost:8529",
+  user: "root",
+  password: ""
+})
+
+const project = new Project({
+  sources: [
+    {
+      name: "schema.graphqls",
+      body: `
+      type Order @rootEntity {
+        orderNumber: String
+          items: [OrderItem]
+          deliveryAddress: Address
+        paymentInfo: PaymentInfo
+      }
+      
+      type PaymentInfo @entityExtension {
+        creditCardNumber: String
+      }
+      
+      type OrderItem @childEntity {
+        itemNumber: String
+          quantity: Int
+      }
+      
+      type Address @valueObject {
+        street: String
+        postalCode: String
+        city: String
+          country: Country @reference
+      }
+      
+      type Country @rootEntity {
+          isoCode: String @key
+          description: String
+      }`
+    },
+    {
+      name: "permission-profiles.json",
+      body: JSON.stringify({
+        permissionProfiles: {
+          default: {
+            permissions: [
+              {
+                roles: ["users"],
+                access: "readWrite"
+              }
+            ]
+          }
+        }
+      })
+    }
+  ],
+  getExecutionOptions: ({ context }) => ({ authRoles: ["users"] }),
+//   getOperationIdentifier: ({ context }) => context as object // each operation is executed with an unique context object
```

Comment:
I would un-comment this; you pass the request as context, so this should be fine

Reply:
right, this is for version 1.0.5 - getOperationIdentifier is not there yet

Reply:
I take this back.. I get this TS error:
```
Argument of type '{ sources: { name: string; body: string; }[]; getExecutionOptions: ({ context }: ExecutionOptionsCallbackArgs) => { authRoles: string[]; }; getOperationIdentifier: ({ context }: { context: any; }) => object; }' is not assignable to parameter of type 'ProjectConfig | SourceLike[]'.
  Object literal may only specify known properties, and 'getOperationIdentifier' does not exist in type 'ProjectConfig | SourceLike[]'.ts(2345)
```

Reply:
added a ts-ignore to get around the typing issue

Reply:
ah of course, I forgot.